### PR TITLE
Add meta description to some pages

### LINKED ISSUE #3852: Add meta descriptions (III)
Hello team, this issue aims to add a meta description to some pages that lack it.

Regards,

Pilar

### PR DESCRIPTION
**Pull request by PilarQuesada**
Hi team, this PR closes #3852 

## Description
I have added a meta description to several pages of the documentation. 

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns. 
- [x] Updated the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

Regards!

### CODE REVIEW

File: source/docker/docker-installation.rst [hunk 1]
```diff
@@ -1,5 +1,7 @@
 .. Copyright (C) 2021 Wazuh, Inc.
-
+.. meta::
+  :description: Check out this section of the Wazuh documentation to learn more about the Docker installation and how to install the Docker engine and the Docker compose.
```
Comment (MarVergara) [inline source/docker/docker-installation.rst:3]:
```suggestion
  :description: Check out this section of the Wazuh documentation to learn about Docker installation: how to install the Docker engine and the Docker compose. 
```

File: source/installation-guide/more-installation-alternatives/elastic-stack/index.rst [hunk 2]
```diff
@@ -1,13 +1,12 @@
 .. Copyright (C) 2021 Wazuh, Inc.
-
+.. meta::
+  :description: Check out how to install Wazuh with Elastic Stack basic license: deployment types, installation methods, and more.
```
Comment (MarVergara) [inline source/installation-guide/more-installation-alternatives/elastic-stack/index.rst:3]:
```suggestion
  :description: Check out how to install Wazuh with Elastic Stack basic license, deployment types, installation methods, and more. 
```

File: source/installation-guide/open-distro/distributed-deployment/step-by-step-installation/wazuh-cluster/wazuh_single_node_cluster.rst [hunk 3]
```diff
@@ -1,6 +1,7 @@
 .. Copyright (C) 2021 Wazuh, Inc.
 
-.. meta:: :description: Learn how to install Elastic Stack for using Wazuh on Debian
+.. meta::
+  :description: Looking for How to install Wazuh and Filebeat server components in a single node Wazuh cluster?Check out this section and learn how.
```
Comment (MarVergara) [inline source/installation-guide/open-distro/distributed-deployment/step-by-step-installation/wazuh-cluster/wazuh_single_node_cluster.rst:4]:
```suggestion
  :description: Looking for how to install Wazuh server components and Filebeat in a single node cluster? Check out this section of our documentation.
```

File: source/installation-guide/open-distro/index.rst [hunk 4]
```diff
@@ -1,14 +1,12 @@
 .. Copyright (C) 2021 Wazuh, Inc.
-
+.. meta::
+  :description: Visit our installation guide to get started with Wazuh. In this section, you can learn how to install the Wazuh server.
```
Comment (MarVergara) [inline source/installation-guide/open-distro/index.rst:3]:
```suggestion
  :description: Visit our Installation guide to get started with Wazuh. In this section, you can learn how to install the Wazuh server. 
```

File: source/installation-guide/wazuh-agent/deployment_variables/deployment_variables.rst [hunk 5]
```diff
@@ -1,6 +1,7 @@
 .. Copyright (C) 2021 Wazuh, Inc.
 
-.. meta:: :description: Learn how to use the deployment variables to install, register and configure Wazuh agents. 
+.. meta::
+  :description: Learn about the variables that facilitate the deployment of the Wazuh agent on Linux, macOS, Windows and AIX operating systems.
```
Comment (MarVergara) [inline source/installation-guide/wazuh-agent/deployment_variables/deployment_variables.rst:4]:
```suggestion
  :description: Learn about the variables that facilitate the deployment of the Wazuh agent on Linux, macOS, Windows, and AIX operating systems.
```

File: source/installation-guide/wazuh-agent/wazuh_agent_package_aix.rst [hunk 6]
```diff
@@ -1,6 +1,7 @@
 .. Copyright (C) 2021 Wazuh, Inc.
 
-.. meta:: :description: Learn how to install the Wazuh agent on AIX
+.. meta::
+  :description: Learn more about how to install or deploy the Wazuh lightweight agent in AIX in this section of our installation guide.
```
Comment (MarVergara) [inline source/installation-guide/wazuh-agent/wazuh_agent_package_aix.rst:4]:
```suggestion
  :description: Learn how to deploy the Wazuh agent on AIX, using deployment variables that facilitate the task of installing, registering, and configuring the agent. 
```

File: source/installation-guide/wazuh-agent/wazuh_agent_package_hpux.rst [hunk 7]
```diff
@@ -1,6 +1,7 @@
 .. Copyright (C) 2021 Wazuh, Inc.
 
-.. meta:: :description: Learn how to install the Wazuh agent on HP-UX
+.. meta::
+  :description: Learn more about how to install or deploy the Wazuh lightweight agent in HP-UX in this section of our installation guide.
```
Comment (MarVergara) [inline source/installation-guide/wazuh-agent/wazuh_agent_package_hpux.rst:4]:
```suggestion
  :description: Learn more about how to successfully install the Wazuh agent on HP-UX systems in this section of our Installation guide.
```

File: source/installation-guide/wazuh-agent/wazuh_agent_package_linux.rst [hunk 8]
```diff
@@ -1,6 +1,7 @@
 .. Copyright (C) 2021 Wazuh, Inc.
 
-.. meta:: :description: Learn how to deploy Wazuh agents on your Linux systems. 
+.. meta::
+  :description: Learn more about how to install or deploy the Wazuh lightweight agent in Linux in this section of our installation guide.
```
Comment (MarVergara) [inline source/installation-guide/wazuh-agent/wazuh_agent_package_linux.rst:4]:
```suggestion
  :description: Learn how to deploy the Wazuh agent on Linux with deployment variables that facilitate the task of installing, registering, and configuring the agent. 
```

File: source/installation-guide/wazuh-agent/wazuh_agent_package_macos.rst [hunk 9]
```diff
@@ -1,6 +1,7 @@
 .. Copyright (C) 2021 Wazuh, Inc.
 
-.. meta:: :description: Learn how to install the Wazuh agent on macOS.
+.. meta::
+  :description: Learn more about how to install or deploy the Wazuh lightweight agent in macOS in this section of our installation guide.
```
Comment (MarVergara) [inline source/installation-guide/wazuh-agent/wazuh_agent_package_macos.rst:4]:
```suggestion
  :description: Learn more about how to successfully install the Wazuh agent on macOS systems in this section of our Installation guide.
```

File: source/installation-guide/wazuh-agent/wazuh_agent_package_solaris.rst [hunk 10]
```diff
@@ -1,6 +1,7 @@
 .. Copyright (C) 2021 Wazuh, Inc.
 
-.. meta:: :description: Learn how to install Wazuh agents on your Solaris system. 
+.. meta::
+  :description: Learn more about how to install or deploy the Wazuh lightweight agent in Solaris 10 i386 in this section of our installation guide.
```
Comment (MarVergara) [inline source/installation-guide/wazuh-agent/wazuh_agent_package_solaris.rst:4]:
```suggestion
  :description: Learn more about how to successfully install the Wazuh agent on Solaris systems in this section of our Installation guide.
```

File: source/user-manual/manager/alert-threshold.rst [hunk 11]
```diff
@@ -1,5 +1,7 @@
 .. Copyright (C) 2021 Wazuh, Inc.
-
+.. meta::
+  :description: The Wazuh manager is the system that analyzes the data received from all registered agents. Find out more about how to define an alert level threshold here.
```
Comment (MarVergara) [inline source/user-manual/manager/alert-threshold.rst:3]:
```suggestion
  :description: The Wazuh manager is the system that analyzes the data received from all registered agents. Find out more about how to define an alert level threshold.
```

File: source/user-manual/reference/ossec-conf/client.rst [hunk 12]
```diff
@@ -1,5 +1,7 @@
 .. Copyright (C) 2021 Wazuh, Inc.
-
+.. meta::
+  :description: Learn more about the client configuration section and its options in this section of the Wazuh user manual.
```
Comment (MarVergara) [inline source/user-manual/reference/ossec-conf/client.rst:3]:
```suggestion
  :description: Learn more about the client configuration section and its configuring options in this section of the Wazuh user manual. 
```

File: source/user-manual/registering/password-authorization-registration.rst [hunk 13]
```diff
@@ -1,5 +1,7 @@
 .. Copyright (C) 2021 Wazuh, Inc.
-
+.. meta::
+  :description: Check out more about registering Wazuh agents and the registration service with password authorization in our documentation.
```
Comment (MarVergara) [inline source/user-manual/registering/password-authorization-registration.rst:3]:
```suggestion
  :description: Check out our documentation to learn about registering Wazuh agents and the registration service with password authorization. 
```

**Review by MarVergara — CHANGES_REQUESTED** [review]

File: source/user-manual/configuring-cluster/index.rst [hunk 14]
```diff
@@ -1,13 +1,13 @@
...
+.. meta::
+  :description: Learn more about the cluster node configuration in Wazuh: labels, Filebeat, Splunk forwarder installation, and more. 
+
 .. _configuring-cluster:
 
 Deploying a Wazuh cluster
 ===========================
 
-.. meta::
-  :description: A complete user manual about how to configure a Wazuh cluster.
```
Comment (PilarQuesada) [inline source/user-manual/configuring-cluster/index.rst:9]: This meta description is way too short. We need a meta description between 120 and 150 characters, @MarVergara

File: source/user-manual/configuring-cluster/index.rst [hunk 15]
```diff
@@ -1,13 +1,13 @@
 .. Copyright (C) 2021 Wazuh, Inc.
 
+.. meta::
+  :description: Learn more about the cluster node configuration in Wazuh: labels, Filebeat, Splunk forwarder installation, and more.
```
Comment (MarVergara) [inline source/user-manual/configuring-cluster/index.rst:4]:
```suggestion
  :description: Learn more about the cluster node configuration in Wazuh: master and worker nodes, labels, Filebeat, Splunk forwarder installation, and more. 
```

File: source/installation-guide/open-distro/distributed-deployment/step-by-step-installation/elasticsearch-cluster/elasticsearch-multi-node-cluster.rst [hunk 16]
```diff
@@ -1,6 +1,7 @@
 .. Copyright (C) 2021 Wazuh, Inc.
 
-.. meta:: :description: Learn how to install Elasticsearch on a multi-node cluster with Wazuh. This deployment type of Wazuh is convenient for large working environments.
+.. meta::
+  :description: Learn more about how to install Elasticsearch in a multi-node cluster for Wazuh in this step-by-step guide.
```
Comment (MarVergara) [inline source/installation-guide/open-distro/distributed-deployment/step-by-step-installation/elasticsearch-cluster/elasticsearch-multi-node-cluster.rst:4]:
```suggestion
  :description: Learn how to install Elasticsearch on a multi-node cluster with Wazuh. This deployment type provides high availability and scalability of the product. 
```

File: source/installation-guide/open-distro/distributed-deployment/unattended/unattended-elasticsearch-cluster-installation.rst [hunk 17]
```diff
@@ -1,5 +1,7 @@
 .. Copyright (C) 2021 Wazuh, Inc.
-
+.. meta::
+  :description: Check out how to install Open Distro for Elasticsearch and Open Distro for Kibana using an automated script.
```
Comment (MarVergara) [inline source/installation-guide/open-distro/distributed-deployment/unattended/unattended-elasticsearch-cluster-installation.rst:3]:
```suggestion
  :description: Check out how to install Open Distro for Elasticsearch and Open Distro for Kibana using a script that automates the installation process.
```

File: source/installation-guide/wazuh-agent/wazuh_agent_package_aix.rst [hunk 18]
```diff
@@ -1,6 +1,7 @@
 .. Copyright (C) 2021 Wazuh, Inc.
 
-.. meta:: :description: Learn how to install the Wazuh agent on AIX
+.. meta::
+  :description: Learn how to deploy the Wazuh agent on AIX, using deployment variables that facilitate the task of installing, registering, and configuring the agent.
```
Comment (MarVergara) [inline source/installation-guide/wazuh-agent/wazuh_agent_package_aix.rst:4]:
```suggestion
  :description: Learn how to deploy the Wazuh agent on AIX using deployment variables that facilitate the task of installing, registering, and configuring the agent. 
```

File: source/installation-guide/wazuh-agent/wazuh_agent_package_hpux.rst [hunk 19]
```diff
@@ -1,6 +1,7 @@
 .. Copyright (C) 2021 Wazuh, Inc.
 
-.. meta:: :description: Learn how to install the Wazuh agent on HP-UX
+.. meta::
+  :description: Learn more about how to successfully install the Wazuh agent on HP-UX systems in this section of our Installation guide.
```
Comment (MarVergara) [inline source/installation-guide/wazuh-agent/wazuh_agent_package_hpux.rst:4]:
```suggestion
  :description: Learn more about how to successfully install the Wazuh agent on HP-UX systems in this section of our Installation Guide.
```

File: source/installation-guide/wazuh-agent/wazuh_agent_package_macos.rst [hunk 20]
```diff
@@ -1,6 +1,7 @@
 .. Copyright (C) 2021 Wazuh, Inc.
 
-.. meta:: :description: Learn how to install the Wazuh agent on macOS.
+.. meta::
+  :description: Learn more about how to successfully install the Wazuh agent on macOS systems in this section of our Installation guide.
```
Comment (MarVergara) [inline source/installation-guide/wazuh-agent/wazuh_agent_package_macos.rst:4]:
```suggestion
  :description: Learn more about how to successfully install the Wazuh agent on macOS systems in this section of our Installation Guide.
```

File: source/installation-guide/wazuh-agent/wazuh_agent_package_solaris.rst [hunk 21]
```diff
@@ -1,6 +1,7 @@
 .. Copyright (C) 2021 Wazuh, Inc.
 
-.. meta:: :description: Learn how to install Wazuh agents on your Solaris system. 
+.. meta::
+  :description: Learn more about how to successfully install the Wazuh agent on Solaris systems in this section of our Installation guide.
```
Comment (MarVergara) [inline source/installation-guide/wazuh-agent/wazuh_agent_package_solaris.rst:4]:
```suggestion
  :description: Learn more about how to successfully install the Wazuh agent on Solaris systems in this section of our Installation Guide.
```

File: source/installation-guide/wazuh-agent/wazuh_agent_package_windows.rst [hunk 22]
```diff
@@ -1,7 +1,7 @@
 .. Copyright (C) 2021 Wazuh, Inc.
 
 .. meta::
-  :description: Visit the installation guide of the Wazuh agent for Windows in this section of our documentation.
+  :description: Learn more about how to successfully install the Wazuh agent on Windows systems in this section of our Installation guide.
```
Comment (MarVergara) [inline source/installation-guide/wazuh-agent/wazuh_agent_package_windows.rst:4]:
```suggestion
  :description: Learn more about how to successfully install the Wazuh agent on Windows systems in this section of our Installation Guide.
```

File: source/user-manual/reference/ossec-conf/client.rst [hunk 23]
```diff
@@ -1,5 +1,7 @@
 .. Copyright (C) 2021 Wazuh, Inc.
-
+.. meta::
+  :description: Learn more about the client configuration section and its configuring options in this section of the Wazuh user manual.
```
Comment (MarVergara) [inline source/user-manual/reference/ossec-conf/client.rst:3]:
```suggestion
  :description: Learn more about client configuration, connection to the manager, and its configuring options in this section of the Wazuh user manual. 
```

**Review by MarVergara — CHANGES_REQUESTED** [review]

File: source/amazon/index.rst [hunk 24]
```diff
@@ -1,13 +1,12 @@
 .. Copyright (C) 2021 Wazuh, Inc.
-
+.. meta::
+  :description: Using Wazuh to monitor AWS? Learn how Wazuh helps to increase the security of AWS infrastructure in this section of our documentation.
```
Comment (santiago-bassett) [inline source/amazon/index.rst:3]: don't use rhetorical questions in our documentation

File: source/docker/docker-installation.rst [hunk 25]
```diff
@@ -1,5 +1,7 @@
 .. Copyright (C) 2021 Wazuh, Inc.
-
+.. meta::
+  :description: Check out this section of the Wazuh documentation to learn about Docker installation: how to install the Docker engine and the Docker compose.
```
Comment (santiago-bassett) [inline source/docker/docker-installation.rst:3]: This section does not show how to install Docker compose. Instead it shows how to install the Wazuh solution using Docker compose for the deployment of the Wazuh manager and central components.

File: source/getting-started/use_cases/index.rst [hunk 26]
```diff
@@ -1,7 +1,7 @@
 .. Copyright (C) 2019 Wazuh, Inc.
 .. meta::
-  :description: Explore some practical examples of the most common use cases of the Wazuh solution in this section of our documentation. 
-
+  :description: Looking to see how the Wazuh capabilities work? Check out some common use cases of Wazuh in this section of our documentation.
```
Comment (santiago-bassett) [inline source/getting-started/use_cases/index.rst:3]: Same here, do not use rhetorical questions

**Review by MarVergara — APPROVED** [review]

File: source/amazon/index.rst [hunk 27]
```diff
@@ -1,13 +1,13 @@
 .. Copyright (C) 2021 Wazuh, Inc.
 
+.. meta::
+  :description: Learn how Wazuh helps to increase the security of AWS infrastructure in this section of our documentation.
```
Comment (MarVergara) [inline source/amazon/index.rst:4]:
```suggestion
  :description: Learn how Wazuh helps to increase the security of AWS infrastructure, monitoring AWS instances and AWS services, in this section of our documentation. 
```

**Pull request merged**